public/locales/en/panel-gettingstartedtour.json

```diff
@@ -170,9 +170,9 @@
       "surfaces": {
         "intro": "Let's take a look at the surface of Mars."
       },
-      "globebrowsing": {
-        "title": "Globe Browsing",
-        "intro": "The feature that enables us to look at maps on planets is called <italic>GlobeBrowsing</italic>.",
+      "globe-imagery-browser": {
+        "title": "Globe Imagery Browser",
+        "intro": "The feature that enables us to look at maps on planets is called <italic>Globe Imagery Browser</italic>.",
```

Comment:
I don't think this should be in the getting started tour? It should probably be reeverted to what it was before :)